src/plugins/data/public/deprecated.ts

```diff
@@ -70,25 +70,30 @@ import {
 } from './query';
 
 /**
- * @deprecated  This import will be removed in v8.0.0.
+ * @deprecated  This import will be removed.
+ * @removeBy 8.1
  */
 const changeTimeFilter = oldChangeTimeFilter;
 /**
- * @deprecated  This import will be removed in v8.0.0.
+ * @deprecated  This import will be removed.
+ * @removeBy 8.1
  */
 const mapAndFlattenFilters = oldMapAndFlattenFilters;
 /**
- * @deprecated  This import will be removed in v8.0.0.
+ * @deprecated  This import will be removed.
+ * @removeBy 8.1
  */
 const extractTimeFilter = oldExtractTimeFilter;
 /**
- * @deprecated  This import will be removed in v8.0.0.
+ * @deprecated  This import will be removed.
+ * @removeBy 8.1
  */
 const convertRangeFilterToTimeRangeString = oldConvertRangeFilterToTimeRangeString;
 
 /**
  * Filter helpers namespace:
- * @deprecated Please import helpers from the package kbn/es-query directly. This import will be removed in v8.0.0.
+ * @deprecated Import helpers from the "@kbn/es-query" package directly instead.
+ * @removeBy 8.1
  */
 export const esFilters = {
```

Comment:
@lizozom I'm getting this deprecation warning in a WIP branch, but this constant does not exist in `@kbn/es-query`... is this known/forthcoming?